Makefile

```
@@ -0,0 +1,35 @@
+.PHONY: setup install check-hooks
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)